docs/source/userguide/plugin_registries.rst

```diff
@@ -0,0 +1,201 @@
+ | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
```

Comment:
This all looks good to me. I really like the auto-generated documentation of help information for terminal commands as well. That's nice! One thing I think we should do for all documentation .rst files in the future is make all distribution statements a dropdown widget. Just looks better in my opinion. Thanks for going over this and tweaking it @jsolbrig. It looks great!